includes/wizards/class-components-demo.php

```diff
@@ -30,6 +30,13 @@ class Components_Demo extends Wizard {
 	 */
 	protected $capability = 'manage_options';
 
+	/**
+	 * Admin Menu Hook Priority when calling 'admin_menu' action hook.
+	 *
+	 * @var int.
+	 */
+	protected $admin_menu_hook_priority = 100;
+
```

Comment:
Do we need one for the components demo? This doesn't render a menu item.

Reply:
It's a hidden page accessible only via url (admin.php?page=newspack-components-demo)